examples/tweet_generator.py

```diff
@@ -0,0 +1,147 @@
+import re
+import textwrap
+from datetime import datetime
+from PIL import Image, ImageFont, ImageDraw, ImageOps
+from pincer import command, Client, Descripted
+from pincer.objects import Message, Embed, MessageContext
+
+
+class Bot(Client):
+
+    @Client.event
+    async def on_ready(self):
+        print(
+            f"Started client on {self.bot}\n"
+            "Registered commands: " + ", ".join(self.chat_commands)
+        )
+
+    @command(
+        name='twitter',
+        description='to create fake tweets',
+        guild=690604075775164437
+    )
+    async def twitter(self, ctx: MessageContext, content: Descripted[str, '...']):
+        await ctx.interaction.ack()
+
+        message = content
+        for text_match, user_id in re.findall(re.compile(r"(<@!(\d+)>)"), message):
+            message = message.replace(text_match, '@' + str(await self.get_user(user_id)))
```

Comment:
```suggestion
            message = message.replace(text_match, "@%s" % await self.get_user(user_id))
```
% is a little better than a fstring or + here imo